libs/crypto/keys_manager_impl.cpp

```diff
@@ -52,66 +52,51 @@ namespace iroha {
     auto sig = iroha::sign(
         iroha::sha3_256(test).to_string(), keypair.pubkey, keypair.privkey);
     if (not iroha::verify(
-        iroha::sha3_256(test).to_string(), keypair.pubkey, sig)) {
+            iroha::sha3_256(test).to_string(), keypair.pubkey, sig)) {
       log_->error("key validation failed");
       return false;
     }
     return true;
   }
 
-  nonstd::optional<iroha::keypair_t> KeysManagerImpl::loadKeys() {
-    // Try to load from local file
-    auto loadFile = [this](auto filename, auto &res) {
-      return [that = this, &filename, &res](auto keypair)
-          ->nonstd::optional<iroha::keypair_t> {
-        std::ifstream file(filename);
-        if (not file) {
-          that->log_->error("Cannot read '" + filename + "'");
-          return nonstd::nullopt;
-        }
-        file >> res;
-        return keypair;
-      };
-    };
+  bool KeysManagerImpl::loadFile(const std::string &filename,
```

Comment:
This function looks very general.  Maybe move it to `libs/` folder? In this case, need to refactor `loadFile` as a free function.